### ISSUE TITLE
Bug with operation 'incr'

### ISSUE THREAD

**Issue by fukainan** [fppt/jedis-mock#155]
It happens while the origin value does not exist
```
@Test
void testJedis() {
    for (int i = 0; i < 5; i++) {
        new Thread(() -> {
            long incr = couponRedis.incr("testKey");
            System.out.println(String.format("thread: %s, after: %s", Thread.currentThread(), incr));
        }).start();
    }
}
```
thread: Thread[Thread-28,5,main], after: 1
thread: Thread[Thread-25,5,main], after: 1
thread: Thread[Thread-26,5,main], after: 1
thread: Thread[Thread-27,5,main], after: 1
thread: Thread[Thread-29,5,main], after: 1

But the code will work
```
@Test
void testJedis() {
    for (int i = 0; i < 5; i++) {
        new Thread(() -> {
            couponRedis.get("testKey");
            long incr = couponRedis.incr("testKey");
            System.out.println(String.format("thread: %s, after: %s", Thread.currentThread(), incr));
        }).start();
    }
}
```
thread: Thread[Thread-29,5,main], after: 1
thread: Thread[Thread-25,5,main], after: 3
thread: Thread[Thread-26,5,main], after: 2
thread: Thread[Thread-28,5,main], after: 4
thread: Thread[Thread-27,5,main], after: 5

**Comment by inponomarev** [comment]
Hi @fukainan! Thanks for the provided test.

Indeed, the concurrent code in Jedis-mock was buggy. But concurrency was already fixed  in https://github.com/atp-mipt/Jedis-mock, and we expect it to be included in the next release

Meanwhile I can reproduce the bug on 0.1.23, but no longer reproduce it on master branch of https://github.com/atp-mipt/Jedis-mock, which is going to be version 1.0.0

I have included an extra test based on your report to jedis-mock test set.

**Comment by inponomarev** [comment]
fixed in 1.0.0
